app/js/controllers/main.js

```diff
@@ -20,7 +24,8 @@ export default class MainController extends BaseController {
 
     const mountNode = document.querySelector('.app-view-container');
     const speechController = new SpeechController();
-    const server = new Server();
+    const settings = new Settings();
+    const server = new Server({ settings });
```

Comment:
Did you make sure that with 2 instances of `Settings`, both will always remain in sync? If I change the value of a setting, will it be automatically reflected on the other instance too?